dapp/src/env.d.ts

```diff
@@ -11,6 +11,8 @@ interface ImportMetaEnv {
   readonly SOROBAN_NETWORK: string;
   readonly PUBLIC_DEFAULT_FEE: string;
   readonly PUBLIC_DEFAULT_TIMEOUT: number;
+  readonly PUBLIC_STORACHA_SING_PRIVATE_KEY: string;
+  readonly PUBLIC_STORACHA_PROOF: string;
```

Comment:
Is this correct to be `PUBLIC_` in the env? We would not want to leak these in the browser and this must stay on the server side.